tools/nvqpp/nvq++.in

```diff
@@ -331,17 +331,23 @@ done
 while [ $# -ne 0 ]; do
 	# Filter out any target-specific options.
 	if [[ $# -gt 1 ]] && [[ -n "${TARGET_CONFIG}" ]]; then
-		if [[ "$1" = --${TARGET_CONFIG}-* ]]; then
-			# Assume always has an additional value
-			TARGET_ARGS="${TARGET_ARGS} $1 $2"
-			shift 2
-			continue
-		elif [[ "$1" = --${TARGET_CONFIG}-*=* ]]; then
-			# Separate the value of argument
+		if [[ "$1" = --${TARGET_CONFIG}-*=* ]]; then
+			# Separate the value of argument. Perform base64 encoding on
+			# the parameter value in case it has spaces. The runtime will be
+			# responsible for decoding the value.
 			param="$1"
-			TARGET_ARGS="${TARGET_ARGS} ${param%=*} ${param#*=}"
+		    ESC2="base64_"$(echo -n ${param#*=} | base64)
```

Comment:
Is it GitHub or the indentation on lines#339, 340, 347, 348 look off?

Reply:
> Is it GitHub or the indentation on lines#339, 340, 347, 348 look off?

Hmm, it appears this file mostly prefers tabs instead of spaces and one of the merges introduced spaces here. I will also change the new lines to use tabs, too. Thanks.

Reply:
Updated in ec4726d4c